src/app/profile/payment-methods/payment-method/payment-method.tpl.html

```diff
@@ -15,28 +15,41 @@
     <div uib-collapse="$ctrl.isCollapsed" role="tabpanel" class="panel-collapse">
       <div class="panel-body">
         <div class="container-fluid">
-          <div class="row">
+          <div class="row panel--payment-info">
 
             <div ng-if="$ctrl.isCard()" class="col-sm-6 col-md-4">
               <p class="section-title mb0">Name on card</p>
               <p>{{$ctrl.model['cardholder-name']}}</p>
             </div>
-
-            <div ng-if="!$ctrl.isCard()">
-              <div class="col-sm-6 col-md-4">
-                <p class="section-title mb0">Bank Name</p>
-                <p>{{$ctrl.model['bank-name']}}</p>
+            <div ng-if="$ctrl.isCard()" class="col-sm-6 col-md-4">
```

Comment:
The two ng-if="$ctrl.isCard()" divs right next to each other? They're two different columns so there isn't a big gap of empty space between the payment info and the edit/delete buttons.